focus-android/app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -71,6 +74,15 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     private lateinit var startupTypeTelemetry: StartupTypeTelemetry
     private var _binding: ActivityMainBinding? = null
     private val binding get() = _binding!!
+    private lateinit var privateNotificationObserver: PrivateNotificationFeature
```

Comment:
Small nit: either name the variable a feature or name the class an observer. 😄

Reply:
Yeah, we talked about this when I first submitted this, but I still haven't decided. 😅 I guess I have to now. 😄

Reply:
Renamed the variable to `privateNotificationFeature`.